gwsumm/plot/range.py

```diff
@@ -264,6 +264,15 @@ def livetime_(t):
         return (4/3. * pi * ts * range ** 3).to('Mpc^3 kyr')
 
     def combined_time_volume(self, allsegments, allranges):
+        empty = [i for i, r in enumerate(allranges) if not len(r.value)]
+        for i in empty[-1:]:
```

Comment:
Is this actually looping? It seems to me it only gets the last element of `empty`. 

Here is and example of what I mean:
```python
>>> a = [1,2,3,4,5,6]
>>> for i in a[-1:]:
...     print(i)
... 
6
```

Reply:
@iaraota Good catch - I just fixed that. The intent is to go through the list in reverse